doc/reliability.rst

```diff
@@ -0,0 +1,40 @@
+``mybinder.org`` status and reliability
+=======================================
+
+This page serves as a source of information for general Binder status
+updates, building/launching statistics, and reliability metrics.
+
+Site Reliability Goals
+----------------------
+
+``mybinder.org`` is a free service running on open-source software. It should
+be treated as "beta" technology. As such, the Binder team chooses reliability
+goals that are generally less than the gold-standard in industry. Below are
+some general reliability goals we aim for.
+
+* GOAL 1
+* GOAL 2
```

Comment:
Can we change this to "We are still working on defining what the exact goals should be. Currently we label it as 'best effort'"? I think ahveing a simple page that shows a condensed view of the grafana dashboards that we could even use as target for status.mybinder.org is so useful that we should get it online before we workout goals that we are happy with.